tools/vsimporter/src/utils/xc2vs.cpp

```diff
@@ -95,7 +95,7 @@ static VCProjectItem* addFileToVSInternal(const PBXFile* file, VCProject& proj,
   // Take into account path override
   bool useRelativePath = true;
   if (itemHint && !itemHint->pathOverride.empty()) {
-    sbAssert(!isVariant, "Unexpected path override for variant file: " + realPath);
+    sbAssertWithTelemetry(!isVariant, "Unexpected path override for variant file: " + realPath);
```

Comment:
>realPath [](start = 86, length = 8)

Could this path contain PII, like a user name, e.g. /mymac/kellymor/projects/ ?  Or is this handled by the call to sbAssertWithTelemetry?  If the latter, then disregard this comment.